src/string-conditions.js

```diff
@@ -1,45 +1,80 @@
 // 1. Use conditional statements to set answerOne to true if STR_ONE is 'Hello'
 const STR_ONE = 'Hello' // eslint-disable-line no-unused-vars
 
-let answerOne
+let answerOne = Boolean
+
+if (STR_ONE === 'Hello') {
+  answerOne = true
+}
 
 // 2. Use conditional statements to set answerTwo to true if STR_TWO is not 'Hello'
 const STR_TWO = 'Goodbye' // eslint-disable-line no-unused-vars
 
-let answerTwo
+let answerTwo = Boolean
+
+if (STR_TWO !== 'Hello') {
+  answerTwo = true
+}
 
 // 3. Use conditional statements to set answerThree to true if STR_THREE is
 // longer than STR_FOUR
 const STR_THREE = 'Hello' // eslint-disable-line no-unused-vars
 const STR_FOUR = 'Good' // eslint-disable-line no-unused-vars
 
-let answerThree
+let answerThree = Boolean
+
+if (STR_THREE.length > STR_FOUR.length) {
+  answerThree = true
+}
 
 // 4. Use conditional statements to set answerFour to true
 // if STR_FIVE starts and ends with the same character, regardless of case
 
 const STR_FIVE = 'Alexandra' // eslint-disable-line no-unused-vars
 
-let answerFour
+let answerFour = Boolean
+
+if (STR_FIVE[0].toLowerCase() === STR_FIVE[STR_FIVE.length - 1].toLowerCase()) {
```

Comment:
you can also look into:

```js
const STR_FIVE_LOWER = STR_FIVE.toLowerCase()
```
and then check the String.endsWith and startsWith methods